java/code/src/com/suse/manager/webui/templates/minion/ansible-path-content.jade

```diff
@@ -6,7 +6,10 @@ script(type='text/javascript').
     window.csrfToken = "#{csrf_token}";
     window.timezone = "#{h.renderTimezone()}";
     window.localTime = "#{h.renderLocalTime()}";
-    window.actionChains = !{actionChains};
+
+if actionChains
```

Comment:
How does this happen? I thought it would be an empty array in that case

Reply:
Apparently, when `actionChains` is empty in the model, then it renders to a whitespace, leading to invalid JS:

```javascript
window.actionChains = ;
```

If you know a better trick, I'm all ears :smile:

Reply:
It would be, if we at least add the `key` in the `result Map`, like we do [here](https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/suse/manager/webui/controllers/AnsibleController.java#L145) --> [that means here](https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/suse/manager/webui/controllers/MinionController.java#L425). Instead in [`Inventories` call](https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/suse/manager/webui/controllers/AnsibleController.java#L162) we are completely missing the `key`

Reply:
In the end I'm starting to incline that this would be likely the best thing: have 2 separate `jade` templates for these pages..

Reply:
Ah of course, I forgot that this is used for both pages.

Reply:
In this case, I think this makes sense as it is. So it's fine by me :+1: